upgrade/upgrade-scripts/upgrade-container

```diff
@@ -422,6 +422,21 @@ function create_upgrade_container() {
 			die "failed to add '/domain0' to container config file"
 	fi
 
+	#
+	# Lastly, we create the log directory and bind mount this into
+	# the container. This is meant as a way for software running in
+	# the container, to share files with software running on the
+	# host; e.g. during upgrade verify, this directory can be used
+	# to store logs, such that they persist after the container is
+	# destroyed.
+	#
+	mkdir -p "$LOG_DIRECTORY" ||
+		die "failed to create directory: '$LOG_DIRECTORY'"
```

Comment:
Since this directory already exists and is created by [this code here](https://github.com/delphix/delphix-platform/blob/develop/files/common/var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml#L123), we probably do not need this

Reply:
Yea, I thought the same thing.. it doesn't fail if the directory exists, though.. so I thought it was fine to leave it.. this is the same lines we previously removed..

Should I throw an error if it doesn't pre-exist instead? I was just concerned that if it didn't already exist for some reason, then we'd fail with an awkward error.. or perhaps it would just silently not bind mount anything..

I'd prefer to create it if it doesn't exist already, or fail with an appropriate error.. not that I expect that to occur, just that I'd like to handle the abnormal case since it's easy to do so..